components/DropDownMenu/DropDownModal.js

```diff
@@ -66,9 +66,9 @@ class DropDownModal extends PureComponent {
   }
 
   emitOnOptionSelectedEvent(option) {
-    if (this.props.onOptionSelected) {
-      this.props.onOptionSelected(option);
-    }
+    const { onOptionSelected } = this.props;
+
+    onOptionSelected && onOptionSelected(option);
```

Comment:
I'm not sure if we have a convention on this, but we should.

I'm fine with both,`if` or ternary conditional callbacks, but we should agree on one of those.

I think I'd prefer `if` more, kinda easier to read

Reply:
This is not ternary, this is just how `&&` works, if both are true, it returns the right hand side, so you can expect it to execute the function passed to it.

I prefer this, obviously, but we can put it to vote in a dev-weekly, I'll add it to the agenda as a tiny point.

Until then, I'll use `if (x) { x() }`

Reply:
safest would be if (_.isFunction(callback)), but we could also set this to requiredProp and resolve the issue that way